docker/filesystems/samba/README.md

```diff
@@ -0,0 +1,28 @@
+# Introduction
+
+Samba is the standard Windows interoperability suite of programs for Linux and Unix.
+
+## Start
+
+To run our docker-compose file, first, we need to create an environment variable called HOST_IP
+
+1.- For Linux: 
+  Open a terminal and run this command: export HOST_IP=<YOUR_IP> \
+2.- For Windows 10
+  You can find it here: * https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10
```

Comment:
Waarom moet dit? Gebruik of `host.docker.internal` of `gateway.docker.internal`, er moeten geen handmatige acties in zitten